unfoldingWord/translationCore
# Now alerting the user if there is no Internet connection when searching for Door43 projects.

#### This pull request addresses:

- Now alerting the user if there is no Internet connection when searching for Door43 projects.
- Fixed multiple bugs with online import failing to delete projects folder when project online import fails.
- Fixed failing OnlineImportWorkflowActions and LocalImportWorkflowActions unit tests.


#### How to test this pull request:
- Turn off your internet connection.
- Go to **Import Online project** and click access internet. An alert should show up saying **Unable to connect to the server. Please check your Internet connection.**
- Try searching door43 project with different search criteria and the alert should be displayed.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3426)
